lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -66,72 +64,39 @@ def generate
             'SWIFT_INCLUDE_PATHS' => '$(inherited) ',
           }.merge(embedded_content_settings)
 
-          @xcconfig = Xcodeproj::Config.new(config)
+          xcconfig = Xcodeproj::Config.new(config)
 
-          @xcconfig.merge!(merged_user_target_xcconfigs)
+          xcconfig.merge!(merged_user_target_xcconfigs)
 
-          generate_settings_to_import_pod_targets
+          generate_settings_to_import_pod_targets(xcconfig)
 
-          XCConfigHelper.add_target_specific_settings(target, @xcconfig)
+          XCConfigHelper.add_target_specific_settings(target, xcconfig)
 
           targets = pod_targets + target.search_paths_aggregate_targets.flat_map(&:pod_targets)
-          XCConfigHelper.generate_vendored_build_settings(target, targets, @xcconfig)
-          XCConfigHelper.generate_other_ld_flags(target, pod_targets, @xcconfig)
+          XCConfigHelper.generate_vendored_build_settings(target, targets, xcconfig)
+          XCConfigHelper.generate_other_ld_flags(target, pod_targets, xcconfig)
 
           # TODO: Need to decide how we are going to ensure settings like these
           # are always excluded from the user's project.
           #
           # See https://github.com/CocoaPods/CocoaPods/issues/1216
-          @xcconfig.attributes.delete('USE_HEADERMAP')
+          xcconfig.attributes.delete('USE_HEADERMAP')
 
           # If any of the aggregate target dependencies bring in any vendored dynamic artifacts we should ensure to
           # update the runpath search paths.
           vendored_dynamic_artifacts = pod_targets.flat_map(&:file_accessors).flat_map(&:vendored_dynamic_artifacts)
 
           symbol_type = target.user_targets.map(&:symbol_type).uniq.first
           test_bundle = symbol_type == :octest_bundle || symbol_type == :unit_test_bundle || symbol_type == :ui_test_bundle
-          XCConfigHelper.generate_ld_runpath_search_paths(target, target.requires_host_target?, test_bundle, @xcconfig) if target.requires_frameworks? || vendored_dynamic_artifacts.count > 0
+          XCConfigHelper.generate_ld_runpath_search_paths(target, target.requires_host_target?, test_bundle, xcconfig) if target.requires_frameworks? || vendored_dynamic_artifacts.count > 0
 
-          @xcconfig
+          xcconfig
         end
 
         #---------------------------------------------------------------------#
 
         protected
 
-        # @return String the SWIFT_VERSION of the target being integrated
```

Comment:
nothing changed here just organized to make those `private` and then `protected`.